custom_components/ha_hatch/light.py

```diff
@@ -35,7 +35,26 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry, asyn
             light_entities.append(LightRestoreIotEntity(coordinator=coordinator, thing_name=rest_device.thing_name))
             light_entities.append(LightRiotClockEntity(coordinator=coordinator, thing_name=rest_device.thing_name))
         elif isinstance(rest_device, RestIot):
-            light_entities.append(LightRiotEntity(coordinator=coordinator, thing_name=rest_device.thing_name))
-            light_entities.append(LightRiotClockEntity(coordinator=coordinator, thing_name=rest_device.thing_name))
+            light_entities.append(
```

Comment:
looks like there are a few identical blocks, if you don't mind could you collapse them using or so they share a elif block?